public/js/data.js

```diff
@@ -25,6 +25,9 @@ function newCircuit(name, id) {
 
 
 function changeCircuitName(name, id = globalScope.id) {
+      if(name == "") {
+        name = "Empty";
+      }
```

Comment:
replace with 

name = name || "Untitled"

Reply:
It is working 👍 @satu0king . Please Review and Merge